### PR TITLE
Replace styles

### PR DESCRIPTION
**Pull request by BarryLev**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

The styles were compiled by webpacker, and the task was about using sprockets for this process.

## Summary of change

The project was reconfigured, so the styles are compiled by sprockets now:
- Styles, images and fonts were replaced from "javascript" directory to "assets" diretory
- Application.js was changed, so it doesn't compile non-existent styles in "javascript" directory
- Webpacker.yml was rewritten, so webpacker doesn't refer to images
- "image_pack_tag" helpers were changed to "image_tag"

## Testing approach

ToDo

## CHECK LIST
- [x]  СI passed
- [ ]  Сode coverage >=95%
- [x]  PR is reviewed manually again (to make sure you have 100% ready code)
- [x]  All reviewers agreed to merge the PR
- [x]  I've checked new feature as logged in and logged out user if needed
- [x]  PR meets all conventions

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@import "components/**/*";
+@import "pages/**/*";
+
+@import "../../../node_modules/bootstrap/scss/bootstrap";
```
Comment (loqimean) [inline app/assets/stylesheets/application.scss:4]: а так взагалі гуд робити? це ти з документації брав?

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+@import "toastr";
```
Comment (loqimean) [inline app/assets/stylesheets/application.scss:5]: тут же норм

File: app/javascript/packs/application.js [hunk 3]
```diff
@@ -16,19 +16,18 @@ require("@nathanvda/cocoon");
...
-import "../stylesheets/application";
+// import "bootstrap";
+// import "@fortawesome/fontawesome-free/js/all";
+// import "../stylesheets/application";
 
 // import 'ajax/result_button';
 import "ajax/checkbox";
 import "plugins/flatpickr";
-import "@fortawesome/fontawesome-free/css/all";
+// import "@fortawesome/fontawesome-free/css/all";
```
Comment (loqimean) [inline app/javascript/packs/application.js:26]: якщо не треба, то видаляй

File: app/javascript/packs/application.js [hunk 4]
```diff
@@ -16,19 +16,18 @@ require("@nathanvda/cocoon");
 // global.$ = global.jQuery = jQuery;
 // window.$ = window.jQuery = jQuery;
 
-import "bootstrap";
-import "@fortawesome/fontawesome-free/js/all";
-import "../stylesheets/application";
+// import "bootstrap";
+// import "@fortawesome/fontawesome-free/js/all";
+// import "../stylesheets/application";
```
Comment (loqimean) [inline app/javascript/packs/application.js:21]: аналочічно, і зверху теж

File: config/application.rb [hunk 5]
```diff
@@ -15,6 +15,7 @@ class Application < Rails::Application
     config.load_defaults 6.1
 
     config.active_job.queue_adapter = :sidekiq
+    config.assets.enabled = true
```
Comment (loqimean) [inline config/application.rb:18]: а це навіщо?

File: config/environments/development.rb [hunk 6]
```diff
@@ -64,7 +64,8 @@
   # Debug mode disables concatenation and preprocessing of assets.
   # This option may cause significant delays in view rendering with a large
   # number of complex assets.
-  config.assets.debug = true
+  config.assets.debug = false
```
Comment (loqimean) [inline config/environments/development.rb:67]: чому так?

File: config/environments/production.rb [hunk 7]
```diff
@@ -30,7 +30,7 @@
   config.public_file_server.enabled = ENV["RAILS_SERVE_STATIC_FILES"].present?
 
   # Compress CSS using a preprocessor.
-  # config.assets.css_compressor = :sass
+  config.assets.css_compressor = :yui
```
Comment (loqimean) [inline config/environments/production.rb:33]: а це що таке?

File: config/initializers/assets.rb [hunk 8]
```diff
@@ -13,4 +13,4 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
-# Rails.application.config.assets.precompile += %w( admin.js admin.css )
+# Rails.application.config.assets.precompile += %w( *.scss )
```
Comment (loqimean) [inline config/initializers/assets.rb:16]: а це для чого?

**Review by mykhailovasylenchuk — APPROVED** [review]

**Review by loqimean — APPROVED** [review]

**Pull request merged**